### DISCUSSION TITLE
Wasn't "OnClickOutside" supposed to be renderless?

### DISCUSSION THREAD

**Discussion by Miguelklappes** [vueuse/vueuse#618] [Q&A]
Hello,

I wanted to ask if the components were supposed to be renderless or not. In my case, it's rendering a `<div>` tag which I can't change with ``` is="vue:li" ```, because it breaks the event. I can't afford to have a `<div>` tag inside my `<ul>`, even though I could make them behave like li tags by using ARIA, but I don't want to do that.

![imagen](https://user-images.githubusercontent.com/74425206/125018247-b9869380-e042-11eb-8a1d-f51a97e77ed8.png)

**Comment by wheatjs** [comment]
Ah some of them we do need to wrap in components, but for those you are right that we should support changing the tag via `is`. Would us supporting that fix this for you?

**Reply by Miguelklappes** [reply]
Hi Wheat, 

Sorry, I don't get what you mean
